CleverPush/CleverPush/Source/CleverPush.m

```diff
@@ -2341,7 +2336,27 @@ + (void)showTopicsDialog:(UIWindow *)targetWindow {
             });
         }];
     }];
-    
+}
+
+#pragma mark - get the topics id if it is already exist in the userdefault
++ (BOOL)selectedTopics{
+    NSArray* topics = [self getSubscriptionTopics];
+    if (!topics || [topics count] == 0) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+#pragma mark - get the default checked topics from the console.
++ (NSMutableArray*)getDefaultCheckedTopicsId{
+    NSMutableArray* selectedTopicIds = [[NSMutableArray alloc] init];
+    for (id channelTopic in channelTopics) {
+        if (channelTopic != nil && ([channelTopic valueForKey:@"defaultUnchecked"] == nil || ![[channelTopic valueForKey:@"defaultUnchecked"] boolValue])) {
```

Comment:
pretty sure you can put [channelTopic valueForKey:@"defaultUnchecked"] in a variable.

Reply:
Yes I will do that

Reply:
interesting

Reply:
maybe newline after channelTopic != nil && ([channelTopic valueForKey:@"defaultUnchecked"] == nil